enterprise/core-edge/src/main/java/org/neo4j/coreedge/raft/replication/tx/ReplicatedTransactionCommitProcess.java

```diff
@@ -111,15 +114,13 @@ transaction content does eventually get replicated (e.g. delayed on the network)
             }
             catch ( InterruptedException | TimeoutException  e )
```

Comment:
Sorry about being unclear. I was just reaffirming the fact that we can't and aren't giving up, but that we should keep the state.